rollup.config.js

```diff
@@ -18,7 +18,7 @@ export default {
   ],
   external(id) {
     return Object.keys(pkg.dependencies)
-      .includes(id.split('/')[0]);
+      .find(dep => id.startsWith(dep));
```

Comment:
Let's click this once for all:

* We want to ensure that dependencies start with `{ DEPENDENCY_NAME}/`, not `{ DEPENDENCY_NAME }`.

Reply:
> We want to ensure that dependencies start with `{ DEPENDENCY_NAME}/`, not `{ DEPENDENCY_NAME }`.

Not sure if I understand you correctly. Won't we need both? We want `@lezer/highlight` to be external, as well as `@lezer/highlight/specific/import`.

internal things should be stuff like `./index`, `/path/to/index.js`, `C:\windows\index.js` and the likes

Reply:
Yes. but we don't necessarily want `@lezer/highlight-too/wooop` to be external, if `@lezer/highlight` is a dependency, but `@lezer/highlight-too` is not.

Reply:
@marstamm To drive this forward, could you share a `console.log` of dependencies passed into this on Windows?

Reply:
Output changes depending on what is internal/external, as the tree traversing stops when it is marked as external. This is how the output looks with the changes in this PR:

```sh
internal: D:\code\lezer-feel\src\index.js
internal: D:\code\lezer-feel\src\index.js
internal: ./parser
internal: ./tokens
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\parser.js
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\parser.js
internal: D:\code\lezer-feel\src\parser.js
internal: ./parser.terms.js
external: @lezer/lr
external: @lezer/lr
internal: ./tokens
internal: ./highlight
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\parser.terms.js
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\tokens.js
internal: D:\code\lezer-feel\src\parser.terms.js
internal: D:\code\lezer-feel\src\parser.terms.js
internal: D:\code\lezer-feel\src\highlight.js
internal: D:\code\lezer-feel\src\highlight.js
internal: D:\code\lezer-feel\src\highlight.js
external: @lezer/highlight
```

Reply:
All right! Let's get this merged as is and wait for https://github.com/nikku/lezer-feel/pull/21#discussion_r1115716300 as an edge case pop up in the future.